performance_test/src/communication_abstractions/ros2_callback_communicator.hpp

```diff
@@ -16,6 +16,7 @@
 #define COMMUNICATION_ABSTRACTIONS__ROS2_CALLBACK_COMMUNICATOR_HPP_
 
 #include <rclcpp/rclcpp.hpp>
+#include <rclcpp/qos.hpp>
```

Comment:
@BMarchi changes for new ROS2 QoS class have already been merged in master : https://github.com/ApexAI/performance_test/pull/88/files